client/my-sites/plugins/plugin-details-CTA/style.scss

```diff
@@ -13,12 +13,25 @@
 }
 
 .plugin-details-cta__dialog-content {
+	.dialog__content {
+		padding: 0;
+	}
+
 	.eligibility-warnings {
 		margin: 0;
 
 		.card {
 			box-shadow: none;
 		}
+
+		.card:last-of-type.is-compact {
+			border-top: 1px solid var(--color-border-shadow);
+		}
+
+		.eligibility-warnings__confirm-buttons {
+			display: flex;
+			justify-content: flex-end;
+		}
```

Comment:
Can remove this entirely once https://github.com/Automattic/wp-calypso/pull/68319/files?w=1#diff-4013ca294ac11b95407707f0c33102157990127a3cdce11e6527eb087b09cb3fR5-R8 is merged.